buf.gen.yaml

```diff
@@ -0,0 +1,48 @@
+version: v1
+managed:
+  enabled: true
+  go_package_prefix:
+    default: github.com/tdex-network/tdex-prototbuf/api-spec/protobuf/gen
+    except:
+      - buf.build/googleapis/googleapis
+plugins:
+  # Golang
+  - remote: buf.build/protocolbuffers/plugins/go
+    out: api-spec/protobuf/gen/go
+    opt: paths=source_relative
+  - remote: buf.build/grpc/plugins/go
+    out: api-spec/protobuf/gen/go
+    opt: paths=source_relative,require_unimplemented_servers=false
+  - remote: buf.build/grpc-ecosystem/plugins/grpc-gateway
+    out: api-spec/protobuf/gen/go
+    opt: paths=source_relative
+  # Python
+  - remote: buf.build/protocolbuffers/plugins/python
+    out: api-spec/protobuf/gen/python
+  - remote: buf.build/grpc/plugins/python
+    out: api-spec/protobuf/gen/python
+  # Java
+  - remote: buf.build/protocolbuffers/plugins/java:v3.19.1-1
+    out: api-spec/protobuf/gen/java
+  - remote: buf.build/grpc/plugins/java:v1.42.1-1
+    out: api-spec/protobuf/gen/java
+  # Java-lite
+  - remote: buf.build/protocolbuffers/plugins/java:v3.19.1-1
+    out: api-spec/protobuf/gen/java-lite
+    opt: lite
+  - remote: buf.build/grpc/plugins/java:v1.42.1-1
+    out: api-spec/protobuf/gen/java-lite
+    opt: lite
+  # JavaScript
+  - remote: buf.build/protocolbuffers/plugins/js:v3.19.1-1
+    out: api-spec/protobuf/gen/js
+    opt:
+    - import_style=commonjs
+    - binary
+  - remote: buf.build/grpc/plugins/node:v1.11.2-1
+    out: api-spec/protobuf/gen/js
+    opt:
+      - import_style=commonjs
+  #OpenApi
+  - remote: buf.build/grpc-ecosystem/plugins/openapiv2
+    out: api-spec/protobuf/tdexswap
```

Comment:
is this very different path than others. Is expected?

Reply:
intention is to put generated swagger files in same location as proto files